cmd/proxy-server/main.go

```diff
@@ -581,6 +594,40 @@ func refreshTokenHandler(client pb.TenantServiceClient, log *logrus.Entry) http.
 	})
 }
 
+// refreshAdminTokenHandler refreshes an admin token
+func refreshAdminTokenHandler(client pb.AuthServiceClient, log *logrus.Entry) http.Handler {
```

Comment:
`client` isn't used in this handler. What is this for?

Reply:
Removed. Missed this when making changes to refresh.